runtime/src/lib.rs

```diff
@@ -299,11 +299,15 @@ impl pallet_sudo::Config for Runtime {
 	type Call = Call;
 }
 
+parameter_types! {
+	pub const MaxDataLength: u32 = 100;
```

Comment:
I'm actually not so sure we need this because there is already another length parameter which is max extrinsic size (or max transaction size) already in the runtime core iirc. Substrate transaction storage I think mentions about it in their test/examples.